welcome/index.adoc

```diff
@@ -281,7 +281,7 @@ xref:../updating/updating-disconnected-cluster.adoc#updating-disconnected-cluste
 
 - **xref:../logging/cluster-logging.adoc#cluster-logging[Work with OpenShift Logging]**: Learn about OpenShift Logging and configure different OpenShift Logging types, such as Elasticsearch, Fluentd, and Kibana.
 
-- **xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[Monitor clusters]**:
+- **xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[monitor clusters]**:
```

Comment:
You could keep "Monitor" as upper case, given that it is the first word in the bulleted item.